better_html.gemspec

```diff
@@ -21,17 +21,17 @@ Gem::Specification.new do |s|
     "allowed_push_host" => "https://rubygems.org"
   }
 
+  s.extensions    = ['ext/better_html_ext/extconf.rb']
```

Comment:
Not sure this is enough to tell Rubygems to build the extension after the gem is released, but I think so.